testsuite/run_sets/github_validation/github_validation_init_clients.yml

```diff
@@ -6,4 +6,5 @@
 - features/github_validation/init_clients/sle_minion.feature
 - features/github_validation/init_clients/min_rhlike_salt.feature
 - features/github_validation/init_clients/min_deblike_salt.feature
+- features/github_validation/init_clients/proxy_container.feature
```

Comment:
In fact, we are not initializing it... but.. in any case, this feature make more sense on top of this list.
As having the proxy correctly working it's a dependency for all the clients, which must be connected thru the proxy.

Reply:
Moved to the top.
As given in my comment above https://github.com/uyuni-project/uyuni/pull/8446#discussion_r1553043951 bootstrapping proxy with minions attached is likely to take more efforts than initially assumed. I would divide it into three separate pull requests:
1. Creating proxy configuration and starting podman containers (this PR)
2. Bootstraping proxy with minions re-attached from server
3. `systemd is-active` related tests readjusted for Proxy in CI

What do you think about it?